gunicorn.config.py

```diff
@@ -0,0 +1,8 @@
+bind = "0.0.0.0:2020"
```

Comment:
```suggestion
bind = "127.0.0.1:2020"
```

We seemed to need the more general thing in order for some stuff running inside docker to be accessible from outside that container, but this is running on the same host as the nginx server so localhost should be fine.

Reply:
Good catch!